pagopa/io-app
# fix(IDPay): [IODPAY-191] InitiativesList scroll

## Short description
moved header and creditCard data component into the scrollview in `AvailableInitiativesList.tsx`

## List of changes proposed in this pull request
- removal of `BaseScreenComponent` from the screen
- addition of `BaseHeader`
- moved everything besides the new `BaseHeader` inside of a scrollview (including BaseHeader adds a visual glitch when scrolling down)

## How to test
navigate to a payment method's screen, tap on "show all initiatives" and make sure scrolling up/down moves the whole screen except for the navigation bar.
